src/server/pps/cmds/cmds.go

```diff
@@ -824,6 +761,75 @@ you can increase the amount of memory used for the bloom filters with the
 	return result, nil
 }
 
+func handleCreatePipeline(metrics bool, reprocess bool, rebuild bool, pushImages bool, registry string, username string, password string, pipelinePath string, update bool) error {
```

Comment:
Minor nit, let's just call this `createPipeline` terser and equally clear imo, if not a bit clearer.

Reply:
This isn't possible since `createPipeline` is the name of the cobra command already